### ISSUE TITLE
suspicious scala-xml dependencies

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#159]
are these outdated scala-xml dependencies in common.conf actually necessary? they aren't commented, so they have the look of something that just got kept by accident. we'd have to investigate

in two places:

```
commands += "set libraryDependencies += \"org.scala-lang.modules\" %% \"scala-xml\" % \"1.0.0-RC6\""
```

**Comment by SethTisue** [comment]
the `browse` one will be eliminated by my draft changes addressing https://github.com/scala/community-builds/issues/300

**Comment by SethTisue** [comment]
and the `scalatest` one got fixed in the 2.12 build when we moved to ScalaTest 3.0.
